tap_dashboard_reports/streams.py

```diff
@@ -3,7 +3,7 @@
 
 from singer_sdk import typing as th
 from singer_sdk import Stream
-from tap_dashboard_reports.query import render_query
+from tap_dashboard_reports.query import render_query, _format_date
```

Comment:
`_` in the beginning means that this is a private method of module. So it's bad practice to import it anywhere

Reply:
erase import and add strftime method to end_date